server/chat-plugins/scavenger-games.ts

```diff
@@ -32,6 +32,13 @@ interface GameMode {
 	[k: string]: any;
 }
 
+function toSeconds(time: string) {
+	// hhmmss => ss
+	const parts = time.split(':').reverse();
+	const seconds = parts.map((value: string, index: number) => parseInt(value) * Math.pow(60, index));
+	return seconds.reduce((a, b) => a + b);
```

Comment:
```suggestion
	return parts
		.map((value: string, index: number) => parseInt(value) * Math.pow(60, index));
		.reduce((a, b) => a + b);
```

Reply:
Also, are the typedefs in the map() callback necessary? Should be implicit.

Reply:
probably not - i probably was just trying to hard convert a string to a number before and be lazy with types